keyboards/duck/jetfire/indicator_leds.c

```diff
@@ -123,7 +110,7 @@ void send_value(uint8_t byte, enum Device device)
 
 void send_color(uint8_t r, uint8_t g, uint8_t b, enum Device device)
 {
-  send_value(g, device);
   send_value(r, device);
+  send_value(g, device);
```

Comment:
Have you tested this?  Because some RGB strips have the rgb out of order.  So this may have been intentional.

Reply:
I currently only have 1 fully QMK powered Duck board in my possession. It works on my Duck Eagle V2. If it is how you have said, I can revert the changes just to be absolutely sure. This change is to make room for the sidewinder/raven/tcv3 that I'll be putting in shortly so I'll find out soon haha. 

What are your thoughts.

Reply:
If that board is using the same sort of code, and it works like this, then it should be fine. 

Otherwise, we can either track people down to test it, or merge and hope for the best, or just revert this one change.

Reply:
We have another confirmation of the octagon working, my confidence about this working has increased. I would say this is a non issue.